client/src/App.js

```diff
@@ -60,6 +61,7 @@ const App = () => {
           <Route path="/resources" exact component={Resources} />
           <Route path="/resources/unknown" component={ResourceUnknown} />
           <Route path="/resources/:id" component={ResourceDetail} />
+          <Route path="/userroles" component={RolesPage} />
```

Comment:
Change url to `/role-approval`.

In addition, should be a `PrivateRoute` to ensure authed.